### DISCUSSION TITLE
Ngrok error No se pudo conectar, prueba otro servidor

### DISCUSSION THREAD

**Discussion by khlaw1** [community/community#68166] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Feature Area

Issues

### Body

Tengo un error con ngrok, me sale: No se pudo conectar, prueba otro servidor. ¿Cómo lo puedo arreglar?, soy nuevo.
![Screenshot_2023-09-25-01-15-44-88_84d3000e3f4017145260f7618db1d683](https://github.com/community/community/assets/128159405/2e44d3c5-c8d8-4b84-9793-0c3a55ecf229)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @khlaw1 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects.

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this Ngrok like their community:https://ngrok.com/slack. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
